package/template-noso-grid/package

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+# Copyright (c) 2022 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(template-noso-grid)
+pkgdesc="Nosometric grid template"
+url=https://github.com/RobotCaleb/noso_template
+pkgver=1.0.0
+timestamp=2022-03-20T18:34Z
+section="templates"
+maintainer="Caleb Anderson <robotrising@gmail.com>"
+license=MIT
+installdepends=(templatectl)
+
+source=("https://github.com/RobotCaleb/noso_template/archive/refs/tags/v${pkgver%-*}.zip")
+sha256sums=(cd7cfcb0c2e9b9734a5e79c00182a4d27858e2c6e2501da54f4d58569171a734)
+
+package() {
+    install -D -m 755 -t "$pkgdir"/usr/share/remarkable/templates \
```

Comment:
These should be installed to opt, templatectl handles getting things into the correct locations.

Reply:
Should be fixed